README.md

```diff
@@ -565,10 +564,10 @@ type webpackImporter = boolean;
 
 Default: `true`
 
-Enables/Disables the default Webpack importer.
+Enables/Disables the default webpack importer.
 
-This can improve performance in some cases. Use it with caution because aliases and `@import` at-rules starting with `~` will not work.
-You can pass own `importer` to solve this (see [`importer docs`](https://github.com/sass/node-sass#importer--v200---experimental)).
+This can improve performance in some cases though use it with caution because aliases and `@import` at-rules starting with `~` will not work.
```

Comment:
remove `though` and use a comma instead

Reply:
the "though" is required to join the two sentences - it's not grammatically correct to omit it, but agree about adding the comma to make it less awkward.